app/routes/index.js

```diff
@@ -13,10 +16,19 @@ export default class IndexRoute extends Route {
       if (response.status === 200 && !userData.incompleteUserDetails) {
         return userData.status ?? defaultStatus;
       } else if (response.status === 401) {
-        alert('You are not logged in. Please login to continue.');
-        window.open(
-          'https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97',
-          '_self'
+        this.toast.warning(
```

Comment:
401 comes under the error category. Can we please use `error` instead of `warning` here?